### DISCUSSION TITLE
If I turn ON manually a switch controlled by the VT integration (a heater) ,  the switch is turned OFF automatically, even with VT in off mode.

### DISCUSSION THREAD

**Discussion by FranMaverix** [jmcollin78/versatile_thermostat#256] [Q&A] [answered]
I don't know is this is a feature or a bug. I didn't could find documentation about it in the integration web.
If it is a feature, it could be configurable ?  If is a bug, how can I fix it?
Thx in advance!

**Comment by jmcollin78** [comment]
Hello,

That is right and this is a feature. For security it is not acceptable to have the VTherm off and the underlying switch on. This could be to overheating the heater. I will not do that, and I suggest you to not do that.

What is the use case ?

**Reply by FranMaverix** [reply, marked as answer]
My heaters are oil radiators that have their own mechanical thermostat. They are set to maximum so that they do not interfere too much with Home Assistant control.
However, these thermostats would prevent overheating in the event of a HA failure or so (they are designed to operate autonomously)

The use case would be that there are elderly people at home who are not too familiar with HA and they only interact with Alexa to turn on the radiators, for example. I don't want them to be cold  :'(

I have renamed the radiators in Alexa App so that it activates VT instead of the radiator, when asked to turn on the radiator.
This solves the problem for now. But it would be interesting to be able to activate/deactivate this VT function, knowing that there is no risk of overheating (in some cases like mine)

Thanks

**Reply by jmcollin78** [reply]
> I have renamed the radiators in Alexa App so that it activates VT instead of the radiator

Yes, it is certainly the best solution. I will certainly not alter the VTherm behavior by adding a mode that could be very disturbing and dangerous for others. Thanks anymway for the idea.

**Reply by FranMaverix** [reply]
That seems fine to me :-)
Really, VT is a great integration and You are HERE giving a marvelous support . That must be valued a lot .
Great work!
